### PR TITLE
Fix datastore types

### PR DESCRIPTION
**Pull request by johnpooch**
@zond apologies. It appears appengine prefers primitive type. Maybe there's another way, but this seems to work.

### CODE REVIEW

**Comment by zond** [comment]
It should work fine with nested structs as well, but you can't let it be a pointer in the datastore structure.

File: game/game.go [hunk 1]
```diff
@@ -442,29 +432,32 @@ type Game struct {
...
-	DisableGroupChat              bool             `methods:"POST,PUT"`
-	DisablePrivateChat            bool             `methods:"POST,PUT"`
-	NationAllocation              AllocationMethod `methods:"POST"`
-	Anonymous                     bool             `methods:"POST"`
-	LastYear                      int              `methods:"POST,PUT"`
-	SkipMuster                    bool             `methods:"POST,PUT"`
-	ChatLanguageISO639_1          string           `methods:"POST,PUT"`
-	GameMasterEnabled             bool             `methods:"POST"`
-	RequireGameMasterInvitation   bool             `methods:"POST,PUT"`
-	DiscordWebhooks               *DiscordWebhooks `methods:"POST" datastore:",noindex"`
```
Comment (zond) [inline game/game.go:467]: Just removing the `*` should make it work I think.

Comment (johnpooch) [reply game/game.go:467]: Done. Tested locally and it worked. thanks!

Comment (johnpooch) [reply game/game.go:467]: But wait, doesn't that mean that it's a required field? I don't want to break existing create game ux

Comment (zond) [reply game/game.go:467]: Go will just default it to the empty value, which in this case will be two empty strings. And datastore works the same way, so nothing will break :D

All old games will just have empty values in those fields.

**Pull request merged**